src/main/resources/assets/ping/lang/en_us.json

```diff
@@ -3,15 +3,15 @@
   "ping:key.categories.ping": "Ping",
 
   "_comment": "Pings",
-  "key.ping": "Opens Ping menu",
-  "ping.key.alert": "Ping (Alert)",
+  "key.ping": "Open pinging menu",
+  "ping.key.alert": "Ping (Attention!)",
```

Comment:
You said some time ago that this type of ping is useful "for making friends aware of danger nearby", so I think word "attention" with an exclamation mark will better fit here than word "alert", but I'm not 100% sure.